src/Facade/Services/ImportBookReportFacadeService.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Linn.Stores.Facade.Services
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Linn.Common.Facade;
+    using Linn.Common.Reporting.Models;
+    using Linn.Common.Reporting.Resources.Extensions;
+    using Linn.Stores.Domain.LinnApps.Reports;
+    using Linn.Stores.Resources.ImportBooks;
+
+    public class ImportBookReportFacadeService : IImportBookReportFacadeService
+    {
+        private readonly IImportBookReportService reportService;
+
+        public ImportBookReportFacadeService(IImportBookReportService reportService)
+        {
+            this.reportService = reportService;
+        }
+
+        public IResult<ResultsModel> GetImpbookIPRReport(IPRSearchResource resource)
+        {
+            DateTime from;
+            DateTime to;
+            try
+            {
+                from = DateTime.Parse(resource.FromDate);
```

Comment:
could do DateTime.TryParse() here instead of a try/catch. Maybe a tad cleaner but I don't have any strong feelings. @richardiphillips likes a TryParse() so he might